src/OpenLoco/src/World/Industry.cpp

```diff
@@ -317,7 +317,7 @@ namespace OpenLoco
         {
             producedCargoMonthlyHistorySize[0]++;
         }
-        const auto newValue = std::min<uint32_t>(producedCargoQuantityMonthlyTotal[0], 12750u) / 50;
+        const auto newValue = static_cast<uint8_t>(std::min<uint32_t>(producedCargoQuantityMonthlyTotal[0], 12750u) / 50);
```

Comment:
What's the point in `std::min` with 12750u if we're just going to throw away the higher bits? Good to make it explicit with the cast, though, I suppose...

Reply:
Its divided by 50